examples/returningPromise.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Legionth\React\Http\HttpServer;
+use React\Socket\Server as Socket;
+use React\Promise\Promise;
+use RingCentral\Psr7\Response;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$callback = function() {
+    return new Promise(function($resolve, $reject) {
+        // Some heavy caluclations
+        sleep(2);
```

Comment:
Did you run a benchmark to confirm this actually works?

This kind of function is exactly what *MUST NOT* be called, see timers instead.